friends/src/App.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import axios from 'axios';
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      friends: [],
+      name: '',
+      age: '',
+      email: '',
+    }
+  }
+
+  componentDidMount() {
+    axios
+    .get('http://localhost:5000/friends')
+    .then(response => {
+    console.log(response);
+    this.setState({friends: response.data})
+    })
+    .catch(err => {
+      console.log(err)
+    })
+  }
+
+  friendName = e => {
+    this.setState({name : e.target.value})
+  }
+
+  friendAge = e => {
+    this.setState({age : e.target.value})
+  }
+
+  friendEmail = e => {
+    this.setState({email : e.target.value})
+  }
+
+  newFriend = () => {
```

Comment:
nice job here.   Remember that age needs to be a number, not a string,  you can use the number method Number() where you pass what you want turned into a number into the function.